### ISSUE TITLE
`pg:backups` public-url does not work without backup ID

### ISSUE THREAD

**Issue by geoffharcourt** [heroku/heroku#1466]
While working on https://github.com/croaky/parity/issues/23, we found we couldn't use the `public-url` without a backup ID and get the URL of the most recent backup as was possible with `heroku pgbackups:url`.

```shell
$ heroku pg:backups public-url --remote production
 !    Resource not found
```

It appears from the [docs](https://devcenter.heroku.com/articles/mapping-pgbackups-commands#downloading-your-backup) that it should be possible to execute public-url without needing to specify a backup ID.

**Comment by lorenjohnson** [comment]
+1

**Comment by zunda** [comment]
+1. Some users want to fetch the latest backup.

**Comment by chibicode** [comment]
:+1:

**Comment by matthewblack** [comment]
:+1:

**Comment by sparestub** [comment]
up

**Comment by kennydust** [comment]
up. no response from heroku team?

**Comment by ktransier** [comment]
:+1:

**Comment by dickeyxxx** [comment]
should be resolved in #1463

**Comment by uhoh-itsmaciek** [comment]
Sorry for the inconvenience. #1463 fixed this. Let us know if you run into any problems.
